### PR TITLE
Make profiles dynamic

### PR DESCRIPTION
**Pull request by queso**
/cc @ryw thoughts?

### CODE REVIEW

**Comment by queso** [comment]
@ryw you feeling ok to merge and release this?

File: client/views/profile.html [hunk 1]
```diff
@@ -4,45 +4,38 @@
...
-          {{> userCard }}
+          <form>
+            {{#each profileFields}}
+              {{> _profileField}}
+            {{/each}}
+            {{submit_button 'Update Profile'}}
+          </form>
         </div>
+
+        {{#if showPreview}}
```
Comment (SantoshSrinivas79) [inline client/views/profile.html:16]: @queso Just tried and it seems to work fine. Except for ... where does {{showPreview}} come from?

**Comment by queso** [comment]
Oh yeah, I should add a setting and hook it up as a helper :)

File: client/views/profile.html (hunk 1 above)
Comment (queso) [inline client/views/profile.html:16]: All fixed, thx @fountainhead!

**Comment by queso** [comment]
@schnie mind giving this a look over?  Would love to merge and release today, but want a second set of internal eyes on it.

**Comment by ryw** [comment]
+1

**Pull request merged**